src/Services/ConvertToJsonService.php

```diff
@@ -46,4 +60,23 @@ public static function unexpectedErrorResponse(): string
         http_response_code(500);
         return json_encode(self::UNEXPECTED_ERROR_RESPONSE);
     }
-}
\ No newline at end of file
+
+    public static function NoTasksAssignedToThatUserErrorResponse(): string
```

Comment:
no capital at start of function

Reply:
NoTasksAssignedToThatUserErrorResponse could it be renamed to noTasksAssignedToUserErrorResponse